scripts/dll/webpackFooterPlugin.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const { validate } = require( 'schema-utils' );
+
+module.exports = class FooterPlugin {
+	/**
+	 * @param {String} footer Text that will be appended to an output file.
+	 */
+	constructor( footer ) {
+		validate( { type: 'string' }, footer, { name: FooterPlugin.name } );
+
+		this.footer = footer;
+	}
+
+	apply( compiler ) {
+		const { ConcatSource } = compiler.webpack.sources;
```

Comment:
This works in webpack v5+. Should we care about webpack v4?

Reply:
We don't. It's a development script used only by us.